code/modules/mob/living/simple_animal/hostile/human/halloween.dm

```diff
@@ -645,3 +645,16 @@
 			timestop(src, duration, magic_range, TRUE)
 			spawn(duration)
 				flags &= ~TIMELESS
+
+
+/mob/living/simple_animal/dracson
+	name = "Young Vampire Bat"
+	desc = "Whatever happened to my Transylvania Twist?"
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "scarybat-dance"
+	mob_property_flags = MOB_UNDEAD | MOB_SUPERNATURAL
+
+/mob/living/simple_animal/death(var/gibbed = FALSE)
```

Comment:
Somehow I don't think this is the right path

Reply:
Oh lord
I moved that guy around so many times deciding where to put him I guess I lost a chunk along the way.